app/src/test/java/org/jellyfin/androidtv/util/DeviceUtilsTest.kt

```diff
@@ -0,0 +1,91 @@
+package org.jellyfin.androidtv.util
+
+import android.os.Build
+import io.mockk.*
+import org.junit.After
+import org.junit.Test
+import org.junit.Assert.*
+
+class DeviceUtilsTest {
+	@After
+	fun tearDown(){
+		unmockkAll()
+	}
```

Comment:
```suggestion
	fun tearDown() = unmockkAll()
```

Reply:
In this case, the built in formatter will revert it back to the original form each time
Can we leave this one as-is to save me going back and changing it each time I touch this file?

Reply:
That doesn't happen to me with the default settings of Android Studio. Are you sure you removed the curly braces?

Reply:
Ah showing my inexperience with Kotlin, didn't realise that was valid syntax :+1: